lstchain/io/event_selection.py

```diff
@@ -178,6 +178,50 @@ def apply_energy_dependent_theta_cuts(self, data, theta_cuts):
         )
         return data[data["selected_theta"]]
 
+    def apply_global_alpha_cut(self, data):
+        """
+        Applying a global alpha cut on a given data
+        """
+        return data[data["alpha"].to_value(u.deg) < self.global_alpha_cut]
+
+    def energy_dependent_alpha_cuts(
+            self, data, energy_bins, min_value=1 * u.deg,
+            max_value=90 * u.deg, smoothing=None, min_events=10
+    ):
+        """
+        Evaluating an optimized energy-dependent alpha cuts, in a given
+        data, with provided reco energy bins, and other parameters to
+        pass to the pyirf.cuts.calculate_percentile_cut function.
+        Note: Using too fine binning will result in too un-smooth cuts.
+        """
+        
+        alpha_cuts = calculate_percentile_cut(
+            data["alpha"],
+            data["reco_energy"],
+            bins=energy_bins,
+            min_value=min_value,
+            max_value=max_value,
+            fill_value=data["alpha"].max(),
+            percentile=100 * self.alpha_containment,
+            smoothing=smoothing,
+            min_events=min_events,
+        )
+        return alpha_cuts
+        
+    def apply_energy_dependent_alpha_cuts(self, data, alpha_cuts):
+        """
+        Applying a given energy-dependent alpha cuts to a data file, along the
+        reco energy bins provided.
+        """
+        
+        data["selected_alpha"] = evaluate_binned_cut(
+            data["alpha"],
+            data["reco_energy"],
+            alpha_cuts,
+            operator.le,
```

Comment:
The operation should be consistent with the operation we use when applying global alpha cuts. I have made a commit to this fix in PR #943. There is no need to apply the minor fix here.

Reply:
ok, thanks for the comment!